Sources/TestingMacros/SuiteDeclarationMacro.swift

```diff
@@ -115,6 +125,8 @@ public struct SuiteDeclarationMacro: MemberMacro, PeerMacro, Sendable {
         diagnostics.append(.availabilityAttributeNotSupported(noasyncAttribute, on: declaration, whenUsing: suiteAttribute))
       }
     }
+
+    return !diagnostics.map(\.severity).contains(.error)
```

Comment:
We tend to avoid `lazy` in the macros target for clarity reasons but yeah, in this case since we're just looking for any matching element, it makes sense here.